src/verifier.cc

```diff
@@ -64,6 +202,10 @@ Result Verifier::Probe(const ProbeCommand* command,
         std::to_string(frame_width) + "," + std::to_string(frame_height) + ")");
   }
 
+  double tolerance[4] = {};
+  bool is_tolerance_percent[4] = {};
+  SetupToleranceForTexels(command, tolerance, is_tolerance_percent);
```

Comment:
Do we need this? It seems like we could do something simple in the for loop below where if the tolerances size > 1 we take tolerance[i % 4] instead of building this array